rviz_default_plugins/src/rviz_default_plugins/tools/select/selection_tool.cpp

```diff
@@ -86,7 +86,7 @@ void SelectionTool::activate()
   context_->getSelectionManager()->setTextureSize(512);
   selecting_ = false;
   moving_ = false;
-//  context_->getSelectionManager()->enableInteraction(true);
+//  context_->getHandlerManager()->enableInteraction(true);
```

Comment:
In all honesty, since this is commented out I think we should just remove the line.  There is no reason to keep dead code around like this in my opinion.

Reply:
Also works for me.